apiary.apib

```diff
@@ -2477,6 +2477,112 @@ Removes also personal data of user (e-mail and name).
 
 + Response 204 (application/json)
 
+## User Metadata [/manage/users/{user_id_or_email}/metadata]
```

Comment:
Doplnit ze sa to da volat aj so session tokenom

Reply:
Mozno by sme mohli zalozit task a pridat to k vsetkym endpointom ktore to podporuju?

Reply:
možná by nebyl task od věci. Dal sem to k tomu headeru že sa može použit session token